src/Surfnet/StepupSelfService/SelfServiceBundle/Tests/Mock/RemoteVetting/MockRemoteVetControllerTest.php

```diff
@@ -124,7 +124,7 @@ public function a_succesful_response_from_a_remote_vetting_idp_should_succeed()
         $c = $this->client->getResponse()->getContent();
         $this->assertEquals(200, $this->client->getResponse()->getStatusCode());
         $this->assertStringStartsWith('https://selfservice.stepup.example.com/second-factor/remote-vetting/match/', $this->client->getRequest()->getUri());
-        $this->assertContains('ss.second_factor.remote_vet_validation.title', $this->client->getResponse()->getContent());
+        $this->assertContains('Controleer informatie', $this->client->getResponse()->getContent());
```

Comment:
To make tests more friendly for international collaborators. Consider using EN as the default language in the controller test.

Reply:
I'm not going to fix it here but added a task to check this later:
https://www.pivotaltracker.com/story/show/171571414

Reply:
Agreed, fyi I slightly changed the task, making it more actionable.